src/annotations/utils/formatQueryString.ts

```diff
@@ -0,0 +1,27 @@
+// Types
+import {GetAnnotationPayload} from 'src/types'
+
+export const formatAnnotationQueryString = (
+  annotation: GetAnnotationPayload,
+  requestType?: string
+): string => {
+  const {stream, start, end, stickers} = annotation
+
+  const getAnnotationParams = new URLSearchParams(
+    JSON.parse(JSON.stringify({start, end}))
+  )
+
+  if (stream) {
+    const streamParam = requestType === 'delete' ? 'stream' : 'streamIncludes'
+
+    getAnnotationParams.append(streamParam, stream)
+  }
+
+  if (stickers) {
+    Object.keys(stickers).forEach((s: string) => {
+      getAnnotationParams.append(`stickerIncludes[${s}]`, stickers[s])
+    })
+  }
+
+  return `?${getAnnotationParams.toString()}`
```

Comment:
I'm not a fan of putting the `?` here. It lowers the reusability of this method. The assumption it makes is that whatever is passed in will always want a `?` prepended to it. This makes it unable to be composed - you always have to pass it the first part or all of the query string. 

```suggestion
  return `${getAnnotationParams.toString()}`
```